transforms/csdl/preprocess_csdl.xsl

```diff
@@ -334,8 +334,10 @@
     </xsl:template>
 
     <!--Remove functions that are blocking beta generation-->
-    <xsl:template match="edm:Schema[@Namespace='microsoft.graph.callRecords']/edm:Function[@Name='getPstnCalls']"/>
-    <xsl:template match="edm:Schema[@Namespace='microsoft.graph.callRecords']/edm:Function[@Name='getDirectRoutingCalls']"/>
+    <xsl:if test="$remove-capability-annotations='True'"><!--Remove functions that are blocking beta generation only for CSDL generation-->
```

Comment:
I think we are overloading the meaning of "remove capability annotations". If it is not too costly, please introduce a new switch.

Reply:
I hear you. I will update the PR again to reflect this.